checks/check.bat

```diff
@@ -0,0 +1,25 @@
+@echo off
+
+call deactivate.bat
+if errorlevel 1 exit 1
+
+@echo on
+
+start /wait "" %cd%\miniforge.exe /InstallationType=JustMe    ^
+                                  /AddToPath=0                ^
+                                  /RegisterPython=0           ^
+                                  /S                          ^
+                                  /D=%cd%\prefix
```

Comment:
Need full path for the target prefix otherwise it doesn't actually install where it should. The full path to the installer executable was thrown in for good measure, but did not appear to be necessary. Also the target prefix needed to be unquoted to work (noted in the docs as well).